mllib-dal/build.sh

```diff
@@ -26,13 +26,24 @@ if [[ -z $CCL_ROOT ]]; then
  exit 1
 fi
 
+if [[ -z $1 ]]; then
+ echo SPARK_VER not defined, using default.
+else
+ SPARK_VER=$1
+fi
+
 echo === Building Environments ===
 echo JAVA_HOME=$JAVA_HOME
 echo DAALROOT=$DAALROOT
 echo TBBROOT=$TBBROOT
 echo CCL_ROOT=$CCL_ROOT
 echo Maven Version: $(mvn -v | head -n 1 | cut -f3 -d" ")
 echo Clang Version: $(clang -dumpversion)
+echo SPARK_VER=$SPARK_VER
 echo =============================
 
-mvn -DskipTests clean package
+if [[ -z $SPARK_VER ]]; then
+ mvn -DskipTests clean package
+else
+ mvn -DskipTests clean package -P$SPARK_VER
```

Comment:
let's put profile options in the beginning, actions in the end.
mvn -P$SPARK_VER -DskipTests .....